gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-common/src/test/java/io/gravitee/am/gateway/handler/common/vertx/web/handler/AuthenticationFlowHandlerTest.java

```diff
@@ -863,22 +1033,25 @@ public void shouldContinue_rememberDevice_with_device_assessment_enabled() throw
     @Test
     public void shouldRedirectToMFAChallengePage_rememberDevice_with_risk_assessment_but_no_device() throws Exception {
         router.route().order(-1).handler(rc -> {
+            EnrollSettings enrollSettings = createEnrollSettings(true, false, MfaEnrollType.REQUIRED, 0, "");
+            ChallengeSettings challengeSettings = createChallengeSettings(true, MfaChallengeType.REQUIRED, "");
```

Comment:
**todo** : based on the test case name, the test is related to risk assesment but the Challenge type is REQUEST, should be RISK_BASED